src/fragments/guides/location-service/setting-up-your-app-js.mdx

```diff
@@ -47,9 +47,13 @@ In the following procedure, you’ll add authentication to your React app using
 
 **To add authentication to your React app**
 
-1. From your projects directory, run the following command, and answer the prompted questions as indicated.
+1. To do so, we'll initialize an Amplify project and add authentication. From your projects directory, run the following command, and answer the prompted questions as indicated.
 
 ```bash
+amplify init
+# Follow the steps to give the project a name, environment name, and set the default text editor.
```

Comment:
I would show each prompt and the recommended answer here

Reply:
@nilisha-jais could we have this added or can you speak with @aspittel to see if we could go without this. I am okay without since the config for init might change based on user's config.

Reply:
Some guides have `init` some don't - most of the good getting started guides have a walk through of `init` ex: https://docs.amplify.aws/lib/datastore/getting-started/q/platform/js/#prerequisites

If you don't want to do this, I would instead have "make sure to have amplify initialized in your project" rather than listing it as a step without more explanation. Or link to further docs on amplify init.